src/components/NavBar.vue

```diff
@@ -71,12 +71,6 @@ const { data: userClaims } = useQuery({
   staleTime: 5 * 60 * 1000, // 5 minutes
 });
 
-const isAdmin = computed(() => {
-  if (userClaims.value?.claims?.super_admin) return true;
-  if (_isEmpty(_union(...Object.values(userClaims.value?.claims?.minimalAdminOrgs ?? {})))) return false;
-  return true;
-});
-
```

Comment:
Can you test this on both admin view and participant view, and Google SSO login vs username/password?